jstests/aggregation/isodates.js

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
No problem! I am sorry to have you re-write them, but I think it will be worth it. Thank you!